Docs/source/building/summit.rst

```diff
@@ -99,7 +98,7 @@ We recommend to store the above lines in a file, such as ``$HOME/warpx.profile``
 
    source $HOME/warpx.profile
 
-Optionally, download and build openPMD-api for I/O:
+Optionally, download and build openPMD-api for I/O (only needed if you did not load our module above):
```

Comment:
Is there an advantage in manually building openPMD-api now? If not, should we remove this part altogether?

Reply:
I thought about this, and just kept it for the record if someone wants to use a newer/older release.